Applications/DataExplorer/DataView/StationTreeView.cpp

```diff
@@ -106,9 +106,10 @@ void StationTreeView::contextMenuEvent( QContextMenuEvent* event )
         QString temp_name;
         QMenu menu;
 
-        if (static_cast<StationTreeModel*>(model())->stationFromIndex(index,
-                                                                      temp_name)->type() ==
-            GeoLib::Station::StationType::BOREHOLE)
+        QAction* setNameAction = menu.addAction("Set name...");
+        connect(setNameAction, SIGNAL(triggered()), this, SLOT(setNameForElement()));
+        if (static_cast<StationTreeModel*>(model())->stationFromIndex(index, temp_name)->type()
```

Comment:
Unless you plan to rewrite the whole station model without changing this specific line it doesn't matter...